release-engineering/role-handbooks/branch-manager.md

```diff
@@ -292,77 +310,143 @@ The following are some ways to determine if the release process was successful:
-   experimental:
-      CONFIG: experimental
-      GO_VERSION: 1.12.1
-      K8S_RELEASE: stable
-      BAZEL_VERSION: 0.26.0
-      UPGRADE_DOCKER: 'true'
-   master:
-      CONFIG: master
-      GO_VERSION: 1.12.1
-      K8S_RELEASE: stable
-      BAZEL_VERSION: 0.23.2
-   '1.16':
-      CONFIG: '1.16'
-      GO_VERSION: 1.12.1
-      K8S_RELEASE: stable
-      BAZEL_VERSION: 0.23.2
-   '1.15':
-      CONFIG: '1.15'
-      GO_VERSION: 1.12.1
-      K8S_RELEASE: stable
-      BAZEL_VERSION: 0.23.2
-   '1.14':
-      CONFIG: '1.14'
-      GO_VERSION: 1.12.1
-      K8S_RELEASE: latest
-      BAZEL_VERSION: 0.21.0
-   '1.13':
-      CONFIG: '1.13'
-      GO_VERSION: 1.11.5
-      K8S_RELEASE: stable-1.13
-    BAZEL_VERSION: 0.18.1
-
-   # Note that the last block i.e. the `1.12` block was removed
+      experimental:
+         CONFIG: experimental
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: stable
+         BAZEL_VERSION: 0.28.1
+         UPGRADE_DOCKER: 'true'
+      master:
+         CONFIG: master
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: stable
+         BAZEL_VERSION: 0.23.2
+      '1.16':
+         CONFIG: '1.16'
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: latest-1.16
+         BAZEL_VERSION: 0.23.2
+      '1.15':
+         CONFIG: '1.15'
+         GO_VERSION: 1.12.7
+         K8S_RELEASE: stable-1.15
+         BAZEL_VERSION: 0.23.2
+      '1.14':
+         CONFIG: '1.14'
+         GO_VERSION: 1.12.5
+         K8S_RELEASE: stable-1.14
+         BAZEL_VERSION: 0.21.0
+      '1.13':
+         CONFIG: '1.13'
+         GO_VERSION: 1.11.5
+         K8S_RELEASE: stable-1.13
+         BAZEL_VERSION: 0.18.1
+
+   # Note that the last block i.e., the `1.12` block was removed
+   ```
+
+4. Create a PR with this change and wait for it to be merged. ([example PR](https://github.com/kubernetes/test-infra/pull/13876))
+
+   **Wait for the `test-infra-push-kubekins-e2e` presubmit to finish (you can [check on prow](https://prow.k8s.io/?job=post-test-infra-push-kubekins-e2e)).**
+
+   **Pull latest from `master` before continuing.**
+
+5. Generate the new release branch jobs:
+
+   ```shell
+   bazel run //experiment:prepare_release_branch
+   ```
+
+6. Search for and update jobs that must be manually bumped
+
+   `grep` for [`manual-release-bump-required`](https://github.com/kubernetes/test-infra/search?q=manual-release-bump-required&unscoped_q=manual-release-bump-required) in [test-infra](https://github.com/kubernetes/test-infra) and duplicate block entries appropriately
+
+7. Update release dashboards in [`testgrid/config.yaml`](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml) ([example commit](https://github.com/kubernetes/test-infra/pull/13882/commits/8aad58b09ae718139b44c70512d2d133d094ed82))
+   - Remove the oldest release `sig-release-<version>-{master,informing,all}` dashboards e.g., `sig-release-1.12-all`
+   - Add dashboards for the current release e.g., `sig-release-1.16-{master,informing,all}`
```

Comment:
This is mildly messy in practice and might warrant a link to an example PR from a past remove/add cycle.

Reply:
The beginning of this step already references a commit which illustrates the dashboard changes: https://github.com/kubernetes/test-infra/pull/13882/commits/8aad58b09ae718139b44c70512d2d133d094ed82

(This is probably much simpler than it previously was because the jobs/dashboard/testgroup relationship has been moved to annotations within the job configs themselves.)